libs/Zend/Db/Statement/Pdo.php

```diff
@@ -225,10 +225,12 @@ public function _execute(array $params = null)
     {
         try {
             if ($params !== null) {
-                return $this->_stmt->execute($params);
-            } else {
-                return $this->_stmt->execute();
+                foreach (array_values($params) as $index => $paramValue) {
```

Comment:
btw does it make sense to add one integration test for this with a null value?

Reply:
👍 was planning to but forgot

Reply:
👍 code looks good otherwise and tested it locally, worked. feel free to merge once you've added a test. Must have been a hard fine